.vscode/c_cpp_properties.json

```diff
@@ -1,12 +1,27 @@
 {
-  "configurations": [
-    {
-      "name": "FreeCAD",
-      "includePath": ["${workspaceFolder}/**"],
-      "cStandard": "c17",
-      "cppStandard": "c++17",
-      "configurationProvider": "ms-vscode.cmake-tools"
-    }
-  ],
-  "version": 4
-}
+    "configurations": [
+        {
+            "name": "Win32",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "configurationProvider": "ms-vscode.cmake-tools"
+        },
+        {
+            "name": "Mac",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "configurationProvider": "ms-vscode.cmake-tools"
+        },
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/usr/include/**"
```

Comment:
Linux, Mac and Win32 are special names recognized by vscode depending on which platform you are on. If you are on linux I think it's safe to include `/usr/include/`. If you use conda it will still search for headers in /usr/include and simply not find the conda libraries there.